load.php

```diff
@@ -137,6 +137,37 @@ function( $module_settings ) {
 	return $modules;
 }
 
+/**
+ * Gets the content attribute for the generator tag for the Performance Lab plugin.
+ *
+ * This attribute is then used in {@see perflab_render_generator()}.
+ *
+ * @since n.e.x.t
+ */
+function perflab_get_generator_content() {
+	$active_modules = perflab_get_active_modules();
+
+	return sprintf(
+		'Performance Lab %1$s; modules: %2$s',
+		PERFLAB_VERSION,
+		implode( ', ', $active_modules )
+	);
+}
+
+/**
+ * Displays the HTML generator tag for the Performance Lab plugin.
+ *
+ * See {@see 'wp_head'}.
+ *
+ * @since n.e.x.t
+ */
+function perflab_render_generator() {
+	$content = perflab_get_generator_content();
+
+	echo '<meta name="generator" content="' . esc_attr( $content ) . '">';
```

Comment:
For better readability can we append **\n** at the end of the tag? and the end tag for the meta. In WordPress core, we follow this format. https://github.com/WordPress/WordPress/blob/master/wp-includes/robots-template.php#L49

`echo '<meta name="generator" content="' . esc_attr( $content ) . '"  />' . "\n";`

Reply:
@mukeshpanchal27 Great point, updated!